source/fundamentals/data-formats/pojo-customization.txt

```diff
@@ -648,42 +650,46 @@ see the following API Documentation:
 - `TypeWithTypeParameters <{+api+}/apidocs/bson/org/bson/codecs/pojo/TypeWithTypeParameters.html>`__
 
 For more information on generics and type parameters, see the
-:`Oracle documentation on Invoking and Instantiating a Generic Type <https://docs.oracle.com/javase/tutorial/java/generics/types.html#instantiation>`__.
+`Java language guide on Invoking and Instantiating a Generic Type <https://docs.oracle.com/javase/tutorial/java/generics/types.html#instantiation>`__.
 
 Enum Type Support
 ^^^^^^^^^^^^^^^^^
 
-The POJO ``Codec`` uses the ``enum`` constant name as the property value by
-default. The following code snippet shows a POJO that contains an ``enum``
-field:
-
-.. code-block:: java
+The ``PojoCodecProvider`` includes a codec to convert the ``enum`` property
```

Comment:
As of 4.5, this is no longer part of the PojoCodecProvider.  Rather, the PojoCodecProvider delegates to the CodecRegistry to find a Codec for any enum class.  

In practice, applications are unlikely to use enum classes outside of the context of POJOs or the upcoming support for Java records.

Reply:
My mistake, I thought the new codec was included. Would a good way to summarize the "Enum Type Support" for 4.5 in this section (within the POJO customization page) be to mention that as of 4.5, you must provide an enum Codec from another provider?

Reply:
No, the enum code is included in the default registry, but independently of the pojo codec.  The pojo codec used to support enums directly.  Now it delegates to the registry.  So if an application happened to register an enum codec in the registry (unlikely, but possible), it will be used by the pojo codec.